src/common/outbound.c

```diff
@@ -2570,11 +2569,9 @@ load_perform_file (session *sess, char *file)
 	tbuf[1024] = 0;
 	while (fgets (tbuf, 1024, fp))
 	{
-		nl = strchr (tbuf, '\n');
-		if (nl == tbuf) /* skip empty commands */
+		tbuf[strcspn (tbuf, "\r\n")] = 0;
```

Comment:
Well that just breaks Unix.

I think you should just ensure files on Windows are written using `\n` only.

Reply:
Ah my mistake. I do still think there is an advantage to simply using `\n` everywhere.